ignite/metrics/accuracy.py

```diff
@@ -116,6 +116,9 @@ class Accuracy(_BaseClassification):
             default, CPU.
 
     Examples:
+        Here, we first declare the metric which is accuracy in this case, then we attach the metric instance to the 
```

Comment:
Maybe we can just say simply, for more info how metric works with engine see the link